app/experimenter/visualization/tests/api/test_views.py

```diff
@@ -206,9 +206,13 @@ def read(self):
 
         mock_open.return_value = File()
         mock_exists.return_value = True
-        probe_set = NimbusProbeSetFactory.create()
+        probe_set_primary = NimbusProbeSetFactory.create()
         experiment = NimbusExperimentFactory.create_with_status(
-            target_status=status, probe_sets=[probe_set]
+            target_status=status, probe_sets=[probe_set_primary]
```

Comment:
You might wanna set up some primary and secondary probesets here to catch both cases above?

Reply:
The next line below this appends secondary probe sets:
`experiment.probe_sets.add(...)`

Unless there is better way of doing this?

Reply:
Oh yes, no, that's right, I didn't read that correctly.  We should probably add some helper methods to the factory for adding primary/secondary probesets, but yes nevermind me that's right 👍